jekyll/_cci2/contexts.md

```diff
@@ -180,8 +183,7 @@ Environment variables are used according to a specific precedence order, as foll
 Environment variables declared inside a shell command `run step`, for example `FOO=bar make install`, will override environment variables declared with the `environment` and `contexts` keys. Environment variables added on the Contexts page will take precedence over variables added on the Project Settings page.
 
 ## Secrets Masking
-
-_Secrets masking is not currently available on self-hosted installations of CircleCI Server_
+{% include feature-cloud.html %}
```

Comment:
@teesloane Sometimes we need to point out a section on a page that is, for example, Cloud only, even though the full doc is for all versions. I stuck this include in as a 'better' way to do that but I imagine you can devise an actually better way using the in-line-with-title badges and tooltips? If not right now, this might be out of scope for this PR but something to add in after?

Reply:
Hmm. We could make a heading include that generates an h2 with a badge to the right of it, sort of like what the heading does - but I agree — might be best for another PR?

Reply:
Yeah good call, I'll remove this change for now and deal with subsections later :-)